
------------------------------ lisp/org-exp.el ------------------------------
diff --git a/lisp/org-exp.el b/lisp/org-exp.el
index 7b1ba51..c91261b 100644
--- a/lisp/org-exp.el
+++ b/lisp/org-exp.el
@@ -1011,6 +1011,7 @@ value of `org-export-run-in-background'."
 \[l] export as LaTeX
 \[p] export as LaTeX and process to PDF
 \[d] export as LaTeX, process to PDF, and open the resulting PDF document
+\[o] export as Latex Outline, process to PDF, an open the resulting PDF document
 \[L] export as LaTeX to temporary buffer

 \[i] export current file as iCalendar file
@@ -1031,6 +1032,7 @@ value of `org-export-run-in-background'."
             (?R org-export-region-as-html nil)
             (?x org-export-as-xoxo t)
             (?l org-export-as-latex t)
+            (?o org-export-as-latex-outline t)
             (?p org-export-as-pdf t)
             (?d org-export-as-pdf-and-open t)
             (?L org-export-as-latex-to-buffer nil)
--
1.6.3.msysgit.0
